### ISSUE TITLE
Callbacks without parentheses fail

### ISSUE THREAD

**Issue by DGJones** [ghiscoding/angular-markdown-editor#20]
The function **_objectFindById_** in the **_runScopeFunction_** is undefined. 

![image](https://user-images.githubusercontent.com/2074498/32813170-0de52d58-c966-11e7-85ea-617689d3506b.png)

**Comment by ghiscoding** [comment]
Can you create a Plunker to reproduce or create a PR to fix it?

**Comment by DGJones** [comment]
[This Plunker](http://plnkr.co/9fx47VZdXq0DvIzPpmNx) is forked from your current one. In it I have removed the parentheses from the on-fullscreen and on-fullscreen-exit callbacks set in the textarea element:

```
<textarea 	name="editor1" class="content-box"
			ng-model="editor1"
			markdown-editor="{'iconlibrary': 'fa', addExtraButtons: true, resize: 'vertical'}"
			on-fullscreen="onFullScreenCallback"
			on-fullscreen-exit="onFullScreenExitCallback"
			rows="10" >
</textarea>
```

If you run this and press the 'Full Screen Preview' button, you will see the full screen preview as expected. However if you check the debug console you will see the error I mention in the original post.
If you now click on the icon to exit full screen, the window will return to it's original size, however it will still be in preview mode. Checking the console will once again show the error.

It looks like the objectFindById function is simply missing from the file. 

I have implemented another way of binding the callbacks using the localized scope and attribute binding. It works for my purposes, but it does not support multiple arguments. I'll share the code so you can have a look.

**Comment by ghiscoding** [comment]
I took pieces of code from my other library ([Angular-Validation](fix #20 callback not working without parenthese)) and it seems that I forgot to copy over a function that gets called when parentheses aren't supplied. Hopefully that is enough to get you going, I haven't done much changes in this project since a while.

**Comment by DGJones** [comment]
Works great, thanks.
